frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/row/action.dart

```diff
@@ -80,8 +81,8 @@ class _ActionCell extends StatelessWidget {
             action.performAction(context);
           }
         },
-        leftIcon: svgWidget(
-          action.iconName(),
+        leftIcon: FlowySvg(
+          action.icon(),
```

Comment:
The `icon()` method can be a getter, but I didn't want to change that in this PR because it's large.